lib/HTTP/Tiny.pm

```diff
@@ -1638,25 +1638,34 @@ sub can_reuse {
 }
 
 # Try to find a CA bundle to validate the SSL cert,
-# prefer Mozilla::CA or fallback to a system file
-sub _find_CA_file {
+# prefer IO::Socket::SSL::default_ca(), Mozilla::CA or fallback to a system file
+sub _find_CA {
     my $self = shift();
 
     my $ca_file =
       defined( $self->{SSL_options}->{SSL_ca_file} )
       ? $self->{SSL_options}->{SSL_ca_file}
-      : $ENV{SSL_CERT_FILE};
```

Comment:
I think this PR will break installations that have set a SSL_CERT_FILE environment variable but do not have IO::Socket::SSL. Is this case even applicable? If so, this should probably be noted in the docs at least.

Reply:
(The patch includes a note in the documentation but maybe it could be made a little more explicit?) I'll post a suggestion there.

Reply:
I'm not aware if there is a case for using HTTP::Tiny with HTTPS without IO::Socket::SSL? 

I'd think that this logic, and the subsequent failover to Mozilla::CA and the list of well known cert store paths could be replaced in favor of IO::Socket::SSL::default_ca

Reply:
If you're going to otherwise do `undef` here then this entire ternary conditional operator becomes redundant, You can just write

```perl
my $ca_file = $self->{SSL_options}{SSL_ca_file};
```